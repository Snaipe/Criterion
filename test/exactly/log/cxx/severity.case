[act]

% log.cc.bin

[assert]

stderr
  -transformed-by replace_version
  equals -contents-of expected/severity.txt
