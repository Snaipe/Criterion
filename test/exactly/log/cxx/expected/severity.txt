[WARN] This is a warning. They indicate some possible malfunction or misconfiguration in the test.
[ERR ] This is an error. They indicate serious problems and are usually shown before the test is aborted.
[====] Synthesis: Tested: 1 | Passing: 1 | Failing: 0 | Crashing: 0 
